src/ic-cdk/src/api/management_canister/schnorr/mod.rs

```diff
@@ -0,0 +1,42 @@
+//! Threshold Schnorr signing API.
+
+use crate::api::call::{call, call_with_payment128, CallResult};
+use candid::Principal;
+
+mod types;
+pub use types::*;
+
+const SIGN_WITH_SCHNORR_FEE: u128 = 26_153_846_153;
```

Comment:
```suggestion
// Source: https://internetcomputer.org/docs/current/references/t-sigs-how-it-works/#fees-for-the-t-schnorr-production-key
const SIGN_WITH_SCHNORR_FEE: u128 = 26_153_846_153;
```
WDYT about giving a source for such a specific magic number?